plugins/background_fetch/configs.h

```diff
@@ -38,7 +38,7 @@ const char PLUGIN_NAME[] = "background_fetch";
 class BgFetchConfig
 {
 public:
-  BgFetchConfig(TSCont cont) : _cont(cont) { TSContDataSet(cont, static_cast<void *>(this)); }
+  explicit BgFetchConfig(TSCont cont) : _cont(cont) { TSContDataSet(cont, static_cast<void *>(this)); }
```

Comment:
What does explicit do?

Reply:
It makes the CppCheck tool in Visual Code happy :). But, it prevents the compiler from doing an implicit conversion to use this constructors. I don't think that can ever happen here, but seems harmless and makes the tool happy :).

Reply:
@SolidWallOfCode any concerns throwing this in here ?

Reply:
No. `explicit` is rarely a problem.  It would likely be the default if the language designers had thought it through more carefully. Too late now.